src/PositionManager.sol

```diff
@@ -259,37 +232,41 @@ contract PositionManager is
     /// @dev Calling increase with 0 liquidity will credit the caller with any underlying fees of the position
     function _increase(
         uint256 tokenId,
-        PositionConfig calldata config,
         uint256 liquidity,
         uint128 amount0Max,
         uint128 amount1Max,
         bytes calldata hookData
-    ) internal onlyIfApproved(msgSender(), tokenId) onlyValidConfig(tokenId, config) {
+    ) internal onlyIfApproved(msgSender(), tokenId) {
+        (PackedPositionInfo info, PoolKey memory poolKey) = getPoolPositionInfo(tokenId);
```

Comment:
maybe im wrong but i imagine it would be decently cheaper to just get these directly from the mappings inline instead of calling a public function

```
PositionInfo info = positionInfo[tokenId];
...
_modifyLiquidity(info, poolKeys[info.poolId()], liquidity.toInt256(), bytes32(tokenId), hookData);
```

Reply:
i do see its neater though, not against keeping it
I'd maybe call it `getPoolAndPositionInfo`

Reply:
either way. I just did it cause it was repeated in 4 funcs